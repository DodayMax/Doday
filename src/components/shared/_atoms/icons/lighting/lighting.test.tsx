import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@components';

describe('Lighting', () => {
  it('renders correctly without props', () => {
    const lighting = shallow(<Icons.Lighting />);

    expect(lighting).toMatchSnapshot();
  });
});