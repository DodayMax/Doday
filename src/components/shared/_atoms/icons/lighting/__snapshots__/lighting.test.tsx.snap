// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lighting renders correctly without props 1`] = `ShallowWrapper {}`;
